go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/fatih/structtag v1.2.0
 	github.com/iancoleman/strcase v0.1.2 // indirect
 	github.com/lyft/protoc-gen-star v0.5.2 // indirect
+	github.com/stretchr/testify v1.7.0
```

Comment:
Please don't add another dependency for testing. Can you use the stdlib testing package?

Reply:
Sorry but for debugging purposes that lib allow to have a meaningful diff... Really useful when doing that kind of tests. If you could allow this one, that would simplify the testing process.